.github/actions/setup-rocm/action.yml

```diff
@@ -56,6 +56,13 @@ runs:
             echo "$msg"
             exit 1
         fi
+    - name: Runner diskspace health check
+      uses: actions/diskspace-cleanup
+      if: always()
+      env:
+          DISKSPACE-CUTOFF: 70
+      with:
+          diskspace-cutoff: "$DISKSPACE-CUTOFF"
```

Comment:
Nit:  I wonder why not just put `diskspace-cutoff: 70` here, or better yet, use 70 as the default vallue for dispace-cleanup action.  Then you can just call

```
- name: Runner diskspace health check
  uses: actions/diskspace-cleanup
  if: always()
```